modules/sdk-core/src/bitgo/inscription/inscription.ts

```diff
@@ -0,0 +1,84 @@
+import assert from 'assert';
+import { IInscription } from './iInscription';
+import { BitGoBase } from '../bitgoBase';
+import { IWallet } from '../wallet';
+import { inscriptions, PreparedInscriptionRevealData, TapLeafScript } from '@bitgo/utxo-ord';
+import { KeyIndices } from '../keychain';
+import { xprvToRawPrv } from '../../account-lib';
+import { HalfSignedUtxoTransaction } from '../baseCoin';
+
+export class Inscription implements IInscription {
+  private readonly bitgo: BitGoBase;
+  private readonly wallet: IWallet;
+
+  constructor(bitgo: BitGoBase, wallet: IWallet) {
+    this.bitgo = bitgo;
+    this.wallet = wallet;
+  }
+
+  async prepareReveal(
+    inscriptionData: Buffer,
+    contentType: string,
+    feeRate: number
+  ): Promise<PreparedInscriptionRevealData> {
+    const user = await this.wallet.baseCoin.keychains().get({ id: this.wallet.keyIds()[KeyIndices.USER] });
+    assert(user.pub);
+    const pubkey = Buffer.from(user.pub);
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore it exists
+    const network = this.bitgo.coin(this.wallet.coin()).network;
```

Comment:
```suggestion
    const network = getNetwork(this.wallet.coin() as V1Network)
```

Reply:
try this? - this should work because we are only using inscriptions with `"bitcoin"` and `"testnet"`

Reply:
ended up writing a utility method to get network based on coin name. If we support doginals getting the network like this ^ wont work 😢